bftengine/include/bftengine/ControlStateManager.hpp

```diff
@@ -53,14 +53,16 @@ class ControlStateManager : public ResPagesClient<ControlStateManager,
   void setEraseMetadataFlag(int64_t currentSeqNum);
   std::optional<int64_t> getEraseMetadataFlag();
 
-  ControlStateManager(IStateTransfer* state_transfer, uint32_t sizeOfReservedPages);
```

Comment:
Looks like as if the `disable` flag is used for testing only; similar the `disable()` function. If that is true then instead of baking the flag into the production code, couldn't we change the test code?
In `simple_test_replica.hpp` we create a `ControlStateManager` only to disable it right away. How about we create a `TestControlStageManager` which implements the same interface? Thereby, we wouldn't need to touch the production ControlStateManager.